src/components/AllWorkOrders.js

```diff
@@ -0,0 +1,193 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTruck, faMapMarkerAlt } from '@fortawesome/free-solid-svg-icons'
+
+import api from '../queries/api'
+import { workOrderFields } from '../queries/fields'
+import { signalsWorkOrderStatuses } from '../constants/statuses'
+
+const fields = workOrderFields.baseFields
+const statuses = signalsWorkOrderStatuses
+
+class AllWorkOrders extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            allWorkOrdersData: [],
+            location: '',
+            currentPage: 1,
+            lastPage: 1,
+        }
+    }
+
+    componentDidMount() {
+        api.allWorkOrders()
+            .getAll()
+            .then(res => {
+                this.setState({
+                    allWorkOrdersData: res.data.records,
+                    lastPage: res.data.total_pages,
+                })
+                console.log(res.data)
+            })
+    }
+
+    handleChange = event => {
+        this.setState({
+            location: event.target.value,
+        })
+    }
+
+    handleSubmit = event => {
+        event.preventDefault()
+        api.allWorkOrders()
+            .searchAll(this.state.location, 1)
+            .then(res => {
+                this.setState({
+                    allWorkOrdersData: res.data.records,
+                    lastPage: res.data.total_pages,
+                    currentPage: 1,
+                })
+                console.log(res.data)
+            })
+    }
+
+    updatePage = pageNumber => {
+        api.allWorkOrders()
+            .searchAll(this.state.location, pageNumber)
+            .then(res => {
+                this.setState({
+                    allWorkOrdersData: res.data.records,
+                    lastPage: res.data.total_pages,
+                    currentPage: pageNumber,
+                })
+                console.log(res.data)
+            })
+        window.scrollTo(0, 0)
+    }
+
+    prevPage = event => {
+        event.preventDefault()
+        // if currentPage !== 1, API call for prev page
+        console.log(typeof this.state.currentPage)
+        if (this.state.currentPage !== 1) {
+            const prevPage = this.state.currentPage - 1
+            this.updatePage(prevPage)
+        }
+        console.log('Prev')
```

Comment:
lots of console.logs. It would be good to remove them or add comments to the code to explain what we're testing in each log.

Reply:
I removed all of the console.logs since these were just part of my workflow for testing along the way. I'll clean these up going forward unless something is ambiguous.